css-media-queries/css-media-queries/meta-viewport.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>CSS Media Queries - Meta Viewport</title>
+    <link rel="stylesheet" href="reset.css" />
+    <style>
+      .container {
+        min-height: 100vh;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      .box {
+        width: 200px;
+        height: 200px;
+        background-color: blue;
+      }
+      @media screen and (min-width: 567px) {
+        .box {
+          width: 200px;
+          height: 200px;
```

Comment:
These properties are redundant since they don't change when the media query is applied